aws/ami/files/scylla_install_ami

```diff
@@ -105,6 +110,27 @@ if __name__ == '__main__':
         if args.repo_for_update:
             run('curl -L -o /etc/apt/sources.list.d/scylla.list {REPO_FOR_UPDATE}'.format(REPO_FOR_UPDATE=args.repo_for_update))
 
+    # use easy_install instead of pip, because easy_install requires
+    # fewer dependencies, no need to install gcc, it makes our AMI smaller.
+    if distro == 'centos':
+        run('yum install -y epel-release')
+        run('yum install -y pystache python-daemon python-setuptools')
+        # Default install prefix on CentOS7 is /usr not /usr/local, but we need
+        # to align path for executable, so specify script-dir to /usr/local/bin
+        run('python -m easy_install --script-dir /usr/local/bin https://s3.amazonaws.com/cloudformation-examples/aws-cfn-bootstrap-latest.tar.gz')
+        pyver = ''
+    else:
```

Comment:
I am merging this, but please future-proof this code by explicitly checking the distribution, and printing out an error for non-supported distributions, @syuu1228.